jsapp/js/components/permissions/userAssetPermsEditor.es6

```diff
@@ -160,6 +160,11 @@ class UserAssetPermsEditor extends React.Component {
       stateObj.formViewDisabled = true;
     }
 
+    // checking `submissionsEdit` implies having `submissionsAdd` checked
+    if (stateObj.submissionsEdit) {
+      stateObj.submissionsAdd = true;
+    }
```

Comment:
The "Add submissions" checkbox should also be greyed out like the other options when "Change submissions" is ticked.

<img width="636" alt="Screen Shot 2021-05-17 at 4 23 20 PM" src="https://user-images.githubusercontent.com/50016008/118568301-68d27880-b72c-11eb-8802-6af2e278a120.png">

Reply:
Looks like this does the trick:

```diff
diff --git a/jsapp/js/components/permissions/userAssetPermsEditor.es6 b/jsapp/js/components/permissions/userAssetPermsEditor.es6
index 7f231c2a5..032079252 100644
--- a/jsapp/js/components/permissions/userAssetPermsEditor.es6
+++ b/jsapp/js/components/permissions/userAssetPermsEditor.es6
@@ -163,6 +163,7 @@ class UserAssetPermsEditor extends React.Component {
     // checking `submissionsEdit` implies having `submissionsAdd` checked
     if (stateObj.submissionsEdit) {
       stateObj.submissionsAdd = true;
+      stateObj.submissionsAddDisabled = true;
     }

     // checking these options implies having `submissionsView` checked
```

Reply:
👍 
Nice catch ;-)